lib/client-assets.php

```diff
@@ -629,3 +629,22 @@ function gutenberg_extend_block_editor_preload_paths( $preload_paths, $post ) {
 	return $preload_paths;
 }
 add_filter( 'block_editor_preload_paths', 'gutenberg_extend_block_editor_preload_paths', 10, 2 );
+
+
+/**
+ * Extends block editor settings to include themeAutoApplyStyles.
+ * This option allows themes to set the default style applied in a block.
+ *
+ * @param array $settings Default editor settings.
+ *
+ * @return array Filtered editor settings.
+ */
+function gutenberg_add_auto_apply_block_styles_setting( $settings ) {
+	$auto_apply_styles = current( (array) get_theme_support( 'editor-default-block-styles' ) );
+	if ( false !== $auto_apply_styles ) {
+		$settings['themeAutoApplyStyles'] = $auto_apply_styles;
```

Comment:
Not sure if it's too late to rename that but I think we should be consistent with naming between `__experimentalPreferredBlockStyles` `editor-default-block-styles` and `themeAutoApplyStyles `.

Maybe `__experimentalPreferredBlockStyles`, `editor-preferred-block-styles` and `preferredBlockStyles`

Reply:
Hi @youknowriad your suggestion was applied 👍